owner-onboarding-server/src/main.rs

```diff
@@ -244,6 +305,7 @@ async fn perform_maintenance(udt: OwnerServiceUDT) -> std::result::Result<(), &'
 
         #[allow(unused_must_use)]
         let (ov_res, ses_res, rtr_res) = tokio::join!(ov_maint, ses_maint, rtr_maint);
+        let win_res = check_registration_window(udt.clone()).await;
```

Comment:
Having this here would mean that this only gets executed after the maintenance or serving job has stopped. This code should probably be executed in the `perform_maintenance` method, and have that take the window into account.

Reply:
I don't know what you mean by 'after the job has stopped'. The `check_registration_window` is called inside the `perform_maintenance` method, as you ask. It simply waits until possible OV registration plus session maintenance has been made.
The window is taken into account inside the `check_registration_window` when we query `fdo_store`.

Reply:
I'm sorry, this was me mis-remembering how many `join!` calls there were in the code: I thought the only one was in the `fn main()`.